r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -471,6 +471,46 @@ test_that("strsplit and str_split", {
   )
 })
 
+test_that("str_to_lower, str_to_upper, and str_to_title", {
+  df <- tibble(x = c("Foo", " B\na R", "ⱭɽⱤoW", "ıI"))
```

Comment:
I've triggered our as-cran build to check this — we might want (or in fact need) to turn these non-ascii characters into their unicode escapes to not anger cran checking (cf https://github.com/apache/arrow/blob/master/r/tests/testthat/test-json.R#L26)

Reply:
It did pass, though https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Encoding-issues recommends using `\u...` for portability.

Reply:
Unicode is tested in C++, so I will remove unicode characters.